src/components/AdminHome/AdminHome.tsx

```diff
@@ -0,0 +1,36 @@
+import { Container, Grid, Typography, Button } from "@material-ui/core";
+import { AdminPanelLinks } from "common/constants";
+import "components/AdminHome/AdminHome.css";
+import SideNav from "components/SideNav/SideNav";
+import { NavLink } from "react-router-dom";
+
+const AdminHome = () => {
+  return (
+    <div className="admin-home">
+      <SideNav />
+      <Container>
+        <Typography variant="h3" style={{ margin: "100px" }}>
+          Administration Panel
+        </Typography>
+        <Grid container spacing={8}>
+          {AdminPanelLinks.map((link) => (
```

Comment:
Usually when iterate to return UI components - JSX elements react recommends adding unique key (mostly performance reasons). You can use index as second argument of the map and pass the index as key to the <Grid item component. In Data Dash the lint checker will have warning on that line. Not a big deal just small improvement

Reply:
done